### PR TITLE
Added new Tile: Blue :D

### PR DESCRIPTION
**Pull request by Beta-King**
http://rawgit.com/Beta-King/labyrinth/master/index.html

### CODE REVIEW

**Comment by Beta-King** [comment]
@abishekvashok @niccokunzmann @samyakbvs please review and merge :D

**Comment by Beta-King** [comment]
screen shot 2018-01-16 at 9 35 15 am --> it works !! :D

**Comment by Beta-King** [comment]
can we merge now ??

File: js/tiles.js [hunk 1]
```diff
@@ -991,6 +991,14 @@ const door = {
             this.ground.show();
         }
     }),
+    blue: Object.assign({}, OpenDoors, {
+        createImages: function() {
+            this.wallTop = this.createImage("tiles/rooms/wall/blueWall.svg");
```
Comment (niccokunzmann) [inline js/tiles.js]: You show a wall there but you use OpenDoors as a base. This means there is a wall but shown but one can walk through it. Would you like to add information about entering and leaving on the top as `false`?

**Comment by niccokunzmann** [comment]
@Beta-King Would it also be ok to merge the master branch, so the build is green again? The syntax seems correct but I would like to make sure by testing.

**Comment by Beta-King** [comment]
wait, i have discovered some problems with this, ill make the changes in a new PR :D thanks for the intel :D

**Comment by Beta-King** [comment]
wait, i have discovered some problems with this, ill make the changes in a new PR :D thanks for the intel :D

**Pull request closed without merging**